site/content/api/Command-And-Control-API.md

```diff
@@ -12,66 +12,52 @@ THIS IS A FIRST DRAFT VERSION OF THE API SPEC AND THUS STILL SUBJECT TO DISCUSSI
 IF YOU THINK THAT THE SPEC SHOULD BE CHANGED OR CAN BE IMPROVED PLEASE DO NOT HESITATE TO OPEN AN ISSUE OR DROP US A LINE ON THE MAILING LIST.
 {{% /warning %}}
 
-A client can be a device connected to Hono directly (using an AMQP connection) or via a *Protocol Adapter* (for example via MQTT,
-LWM2M, etc). Devices and *Protocol Adapters* can also poll Hono periodically in order check for new command & control messages without
-maintaining a constant connection to Hono infrastructure.
+The Command & Control API is defined by means of AMQP 1.0 message exchanges, i.e. a client needs to connect to Hono using AMQP 1.0 in order to invoke operations of the API as described in the following sections. Throughout the remainder of this page we will simply use AMQP when referring to AMQP 1.0.
 
-The Command & Control API is defined by means of AMQP 1.0 message exchanges, i.e. a client needs to connect to Hono using an AMQP 1.0 client in 
-order to invoke operations of the API as described in the following sections.
+# Operations
 
-# Southbound Operations
+The following API defines operations that can be used by *Business Applications* to send commands to devices over an AMQP 1.0 Network and get a response in return.
 
-The following operations can be used by *Devices* and/or *Protocol Adapters* to receive command & control messages sent by *Business Applications*.
+## Send a Command
 
-## Receive Control Message
+### Preconditions
 
-**Preconditions**
+1. *Business Application* has established an AMQP connection with the AMQP 1.0 Network.  
+2. *Business Application* has established an AMQP link in role *sender* with the target address `control/${tenant_id}/${device_id}`. Where `${device_id}` is the ID of the device to send messages to and `${tenant_id}` is the ID of the tenant that the device belongs to.
+3. *Business Application* has established an AMQP link in role *receiver* with the source address `control-reply/${tenant_id}/${device_id}`. Where `${device_id}` is the ID of the device to receive the messages from and `${tenant_id}` is the ID of the tenant that the device belongs to.
```

Comment:
this sounds like the reply-to address is fixed, which FMPOV will not work in a scenario where multiple applications want to send a command to the same device because the response(s) will be delivered to *any one* of the applications (*balanced* message distribution). IMHO the client should therefore use an individual reply-to address that contains an element of uniqueness, e.g. a UUID, as we demand for the other request response APIs.

Reply:
That's right and I do not see a reason to not use a UUID. I think in the beginning with potential queuing for a longer time we had an other expectation but now this fits well.

Reply:
So, does this mean that you are going to update this sentence accordingly?